Jenkinsfile_CNP

```diff
@@ -102,13 +113,16 @@ withPipeline(type, product, component) {
         env.TEST_DOCUMENT_MANAGEMENT_URL = 'http://dm-store-aat.service.core-compute-aat.internal'
         env.CORE_CASE_DATA_API_URL = "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
         env.PREVIEW_BRANCH_WITH_LOCAL_CCD = false;
-        if (env.BRANCH_NAME.startsWith("PR")) {
-            def githubApi = new GithubAPI(this)
+        if (env.BRANCH_NAME.startsWith("PR") ) {
             for (label in githubApi.getLabelsbyPattern(env.BRANCH_NAME, "pr-values:ccd") ) {
                 env.CORE_CASE_DATA_API_URL = "https://ccd-data-store-api-sscs-tribunals-api-pr-${CHANGE_ID}.preview.platform.hmcts.net"
                 println """After akschartsinstall Using CORE_CASE_DATA_API_URL: ${env.CORE_CASE_DATA_API_URL}""".stripIndent()
                 env.PREVIEW_BRANCH_WITH_LOCAL_CCD = true;
```

Comment:
are these two variables doing the same thing: PREVIEW_BRANCH_WITH_LOCAL_CCD, LOCAL_CCD_ENABLED?

Reply:
PREVIEW_BRANCH_WITH_LOCAL_CCD gets its value based on pr-values:ccd tag, whereas LOCAL_CCD_ENABLED (now PREVIEW_CCD_ENABLED) depends on enable_hearings label